Dockerfile

```diff
@@ -46,8 +44,8 @@ RUN pip install "$(printf '%s' ".[${ENVIRONMENT}]")"
 
 # Conditionally install OpenTelemetry packages if USE_OTEL is true
 RUN if [ "$USE_OTEL" = "true" ]; then \
-        pip install opentelemetry-distro opentelemetry-exporter-otlp && \
-        opentelemetry-bootstrap -a install; \
+    pip install opentelemetry-distro opentelemetry-exporter-otlp && \
```

Comment:
Any reason for removing the indentation? I know most shells don't care about indentation in blocks, but from a readability perspective I think it helps.

Reply:
hmm I didn't make any intentional changes; looks like this was done automatically with the Docker formatter. Are you using another formatter?

Reply:
It looks like mine was just the default Docker formatter that comes with VS Code. I was able to turn it off for dockerfiles and add the indentations back.

Reply:
I'm actually intrigued by the idea of linting on Docker files and shell scripts as well (we have a few of both of those).  I don't want to hold up this change, but if you have any information on what formatter VS Code uses under the hood let me know.

Reply:
I looked into it and there is no default Docker formatter/linter; when I saved, the default whitespace and tabspace settings in VS Code were being applied. That said, there are some formatters available for Docker. I haven't done a comprehensive look but [hadolint](https://github.com/hadolint/hadolint) look promising for Dockerfiles.